### DISCUSSION TITLE
Modify list of spells to interrupt

### DISCUSSION THREAD

**Discussion by phoenixsoap** [Hekili/hekili#886] [Q&A] [answered]
Hi, is it possible to modify the list of enemy powers that Hekili uses to recommend interrupts? thanks!

**Comment by Hekili** [comment, marked as answer]
Not at this time.  It's definitely a good idea, though.

**Reply by phoenixsoap** [reply]
alright, thanks for the response!

**Comment by phoenixsoap** [comment]
If you would consider removing "Smite" from the hexxers in De Other Side that would be helpful. I always seem to itchy-trigger-finger that and then miss out on one of the non-spam-cast powers like heal or hex due to the cooldown
